.github/workflows/dev-mode-layer-publish.yml

```diff
@@ -68,5 +68,4 @@ jobs:
             --bucket-name sls-layers-registry \
             --layer-basename sls-external-extension \
             --version $VERSION \
-            --layer-filename go/packages/dev-mode/dist/extension.zip \
-            --github-tag $TAG
+            --layer-filename go/packages/dev-mode/dist/extension.zip
```

Comment:
Why `--github-tag` was removed?

Reply:
I could have this wrong but I thought that the `--github-tag` flag only needed when we wanted to create a release. Since we were not creating a package I didn't think we needed to do a release? 🤔

Reply:
True, it may not work if github release is not created. Ok let's skipt it

Reply:
I can confirm the s3 registry was updated 😎